.github/pull_request_template.md

```diff
@@ -0,0 +1,16 @@
+<!--
+
+
+  (ﾉ◕ヮ◕)ﾉ*:・ﾟ✧
+  
+  Thank you for your interest in contributing to the k6 project!
+  
+  Before you get started, we'd kindly like to ask you to read our:
+    - Contribution guidelines at https://github.com/loadimpact/k6/blob/master/CONTRIBUTING.md
+    - Code of Conduct at https://github.com/loadimpact/k6/blob/master/CODE_OF_CONDUCT.md
+    
+  Out of respect for your time, please start a discussion regarding any bigger contributions
+  either in the #contributors channel of the k6 slack or in the Github Issue before you get
+  started on the implementation.
+  
```

Comment:
Maybe add a sentence at the end, something like
> If you've already done all of that, please go ahead and submit your PR and thank you for the contribution!

Doesn't sound quite right, but :man_shrugging:

Reply:
See proposed change 😄